metafix/src/test/resources/org/metafacture/metafix/integration/lookup/fromXml/toJson/lookupInDeeplyNestedArrayOfObjects/test.fix

```diff
@@ -3,25 +3,24 @@ put_map("rswk-indicator", s: "SubjectHeading")
 set_array("subject[]")
 
 if exists("6890?")
-  set_array("subject[].$append.type[]", "ComplexSubject")
-  set_array("subject[].$last.label")
-  set_array("subject[].$last.componentList[]")
+  set_array("subject[].$append.componentList[]")
```

Comment:
> but tweaked ([...], integration test for field order)

Sorry again. Looks like I accidentally sorted the expected output. Fixed in fe63bad.

Reply:
> The original expected output was identical to the `fromJson` test, so maybe that's where that came from.

Yeah, in the `fromJson` test the input is sorted as well, while in the `fromXml` test the output is generated in the Fix.